SphericalShell/SphericalShell.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Feb  8 08:16:12 2024
+
+@author: Anupama Rajendra
+"""
+import openmc
+import numpy as np
+#import tkinter as tk
+
+# Create materials & export to XML:
+#Simulating tungsten shell:
+W = openmc.Material(name='W_Shell')
+W.set_density('g/cm3', 19.28)
+W.add_element('W', 1.0)
+materials = openmc.Materials([W])
+materials.export_to_xml()
+
+# Create geometry
+#Spherical shell:
+R_1= openmc.Sphere(r=1000) #sphere of radius 1000cm
+inside_sphere_1 = -R_1
+outside_sphere_1 = +R_1
+R_2 = openmc.Sphere(r=1005, boundary_type='vacuum')
+inside_sphere_2 = -R_2
+outside_sphere_2 = +R_2
+R_3 = outside_sphere_1 & inside_sphere_2 #filled with tungsten
+
+# Mapping materials to geometry:
+Void = openmc.Cell(fill=None, region = inside_sphere_1)
+Shell = openmc.Cell(fill=W, region=R_3)
+geometry = openmc.Geometry([Void, Shell])
+geometry.export_to_xml()
+
+
+# # Source distribution:
+PointSource = openmc.stats.Point(xyz=(0.0, 0.0, 0.0))
+Prob = openmc.stats.Discrete(14E+06, 1.0)
+
+# Assign simulation settings
+settings = openmc.Settings()
+settings.batches = 10
+settings.inactive = 1
+settings.particles = 100000
+settings.source = openmc.Source(space=PointSource, energy=Prob, strength = 10.0, particle = 'neutron')
```

Comment:
Is there a reason for `strength = 10`?  It doesn't really matter in this problem, but just curious why not 1 or something larger to represent a realistic system?

Reply:
There's no particular reason for that number. I think it makes to change the source strength to the default 1.0 so that the flux units are still in terms of per-source-particle.

```suggestion
settings.source = openmc.Source(space=PointSource, energy=Prob, strength = 1.0, particle = 'neutron')
```